src/main/java/com/minecolonies/sounds/CitizenSounds.java

```diff
@@ -0,0 +1,82 @@
+package com.minecolonies.sounds;
+
+import com.minecolonies.entity.EntityCitizen;
+import com.minecolonies.lib.Constants;
+import com.minecolonies.util.SoundUtils;
+import net.minecraft.world.World;
+
+/**
+ * Class used to store the basic citizen sounds.
+ */
+public final class CitizenSounds
+{
+    /**
+     * Chance to say a phrase.
+     */
+    private static final int PHRASE_CHANCE = 50;
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private CitizenSounds()
+    {
+        /*
+         * Intentionally left empty.
+         */
+    }
+    /**
+     * The citizen sound events for the females.
+     */
+    public static final class Female
+    {
+        public static final String say = Constants.MOD_ID + ":mob.citizen.female.say";
+
+
+        /**
+         * Private constructor to hide the implicit public one.
+         */
+        private Female()
+        {
+            /*
+             * Intentionally left empty.
+             */
+        }
+    }
+
+    /**
+     * The citizen sound events for the males.
+     */
+    public static final class Male
+    {
+        public static final String say = Constants.MOD_ID + ":mob.citizen.male.say";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00115)